server.js

```diff
@@ -1,10 +1,94 @@
-const express = require('express'); // remember to install your npm packages
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
 
 const server = express();
 
-// add your server code
+server.use(bodyParser.json());
+
+const Budget = require('./budget/budget');
+const Category = require('./category/category');
+const Expense = require('./expense/expense');
+
+mongoose
+  .connect('mongodb://localhost/BudgetTracker')
+  .then(connect => {
+    console.log('API Server Connected');
+  })
+  .catch(error => {
+    console.error('Db connection failed');
+  });
+
+server.get('/', (req, res) => res.send('API Running...'));
+
+server.post('/budgets', (req, res) => {
+  const { title, budgetAmount } = req.body;
```

Comment:
Since we're not using `title` and `budgetAmount` any where, we don't have to destructure the `req.body` object. Instead, just create your new Budget by passing `req.body`:

```JavaScript
const budget = new Budget(req.body);
```